src/openfl/display/Stage.hx

```diff
@@ -1920,6 +1920,18 @@ class Stage extends DisplayObjectContainer implements IModule {
 			
 		}
 		
+		if (__contentsScaleFactor != window.scale && __renderer != null) {
+			
+			__contentsScaleFactor = window.scale;
+			
+			#if (js && html5)
```

Comment:
Why the `#if` fencing? Shouldn't we do the same for cairo, etc?

Reply:
I agree, but I think this is done because we have not tested or focused on Cairo much and have not changed all Cairo renderers to `pixelRatio` yet. @NenadBojkovski, can you confirm this?

For example I found this one here: https://github.com/innogames/openfl/pull/53/files#diff-c1a31d088708f6d62a0bd6ffa0a4fa16

```
#if (js && html5)
    CanvasRenderer.scale = stage.window.scale;
#end
```

it was changed to:

```
#if (js && html5)
    renderSession.pixelRatio = stage.window.scale;
#end
```

That's why the fendinc fencing is there right?

Reply:
Yes we currently are using pixelRatio only for html5 and js target and that's way is fenced. Once we(or someone else) implement support for cairo we can remove it